SPECS-EXTENDED/fstrm/fstrm.spec

```diff
@@ -0,0 +1,117 @@
+%global _hardened_build 1
+%{!?_pkgdocdir: %global _pkgdocdir %{_docdir}/%{name}-%{version}}
+
+Name: fstrm
+Summary: Frame Streams implementation in C
+Version: 0.6.1
+Release: 1%{?dist}
+License: MIT
+URL: https://github.com/farsightsec/fstrm
+Source0: https://dl.farsightsecurity.com/dist/fstrm/fstrm-%{version}.tar.gz
+BuildRequires: autoconf automake libtool
+BuildRequires: libevent-devel
+
+%description
+Frame Streams is a light weight, binary clean protocol that allows for the
+transport of arbitrarily encoded data payload sequences with minimal framing
+overhead -- just four bytes per data frame. Frame Streams does not specify
+an encoding format for data frames and can be used with any data serialization
+format that produces byte sequences, such as Protocol Buffers, XML, JSON,
+MessagePack, YAML, etc.
+
+%package devel
+Summary: Development Files for fstrm library
+Requires: %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+The fstrm-devel package contains header files required to build an application
+using fstrm library.
+
+%package doc
+Summary: API documentation for fstrm library
+BuildArch: noarch
+BuildRequires: doxygen
+Requires: %{name} = %{version}-%{release}
+
+%description doc
+The fstrm-doc package contains Doxygen generated API documentation for
+fstrm library.
+
+%prep
+%setup -q
+# regenerated build scripts to:
+# - remove RPATHs
+# - allow dynamic linking and execution of 'make check'
+autoreconf -fi
+
+%build
+%configure --disable-static --prefix=/tmp/
+make %{?_smp_mflags}
+make html
+
+%install
+# install the library
+make install DESTDIR=%{buildroot}
+rm %{buildroot}%{_libdir}/libfstrm.la
+
+# install documentation
+mkdir -p %{buildroot}%{_pkgdocdir}/
+cp -ar html %{buildroot}%{_pkgdocdir}/html
+
+%check
+make check
+
+%post -p /sbin/ldconfig
+
+%postun -p /sbin/ldconfig
+
+%define _unpackaged_files_terminate_build 0
+
+%files
+%doc COPYRIGHT LICENSE
+%exclude %{_pkgdocdir}/html
+%{_libdir}/libfstrm.so.*
+
+%files devel
+%doc README.md
+%{_bindir}/fstrm_capture
+%{_bindir}/fstrm_dump
+%{_includedir}/fstrm.h
+%{_includedir}/fstrm/
+%{_libdir}/pkgconfig/libfstrm.pc
+%{_libdir}/libfstrm.so
+
+%files doc
+%doc %{_pkgdocdir}/html
+
+%changelog
+* Thu Feb 23 2023 Andy Zaugg <azaugg@linkedin.com> - 0.6.1-1
+- First version of fstrm for cm2 Spec file imported from Centos EPEL
```

Comment:
In order to pass our license checks, you need to specifiy to license of the distro in the changelog. It should match the format specified here: https://github.com/microsoft/CBL-Mariner/blob/2.0/toolkit/scripts/spec_source_attributions.py

Reply:
OK, I think I got it now 😄